packages/angular-cli/tsconfig.json

```diff
@@ -25,9 +25,7 @@
     }
   },
   "exclude": [
-    "blueprints/*/files/**/*"
-  ],
-  "includes": [
-    "./custom-typings.d.ts"
+    "blueprints/*/files/**/*",
+    "**/*.spec.ts"
```

Comment:
We'll move acceptance specs into this package later on. I can remove if you want.